homeassistant/components/notify/llamalab_automate.py

```diff
@@ -0,0 +1,62 @@
+"""
+LlamaLab Automate notification service.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/notify.llamalab_automate/
+"""
+import logging
+import requests
+import voluptuous as vol
+
+from homeassistant.components.notify import BaseNotificationService
+from homeassistant.const import (CONF_API_KEY, CONF_NAME, CONF_PLATFORM)
+
+_LOGGER = logging.getLogger(__name__)
+_RESOURCE = 'https://llamalab.com/automate/cloud/message'
+
+CONF_TO = 'to'
+CONF_DEVICE = 'device'
+
+PLATFORM_SCHEMA = vol.Schema({
```

Comment:
I'm sorry, I should have been a bit more extensive in my feedback. Please extend the notify platform schema. You won't have to specify CONF_PLATFORM and CONF_NAME, those are taken care off. Also, please use `cv.string` for strings

```python
from homeassistant.components.notify import PLATFORM_SCHEMA
from homeassistant.helpers import config_validation as cv

PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
    vol.Required(CONF_API_KEY):  cv.string,
    … etc …
})
```